scripts/release_build_oak_functions

```diff
@@ -1,14 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
This is used in our internal cloudbuild script for release. Is there a plan for updating that as well?

Reply:
Created cl/430414831 as a minimal update to internal references. There's more details in the CL description, we can discuss it there. :)

Reply:
Thanks :)